pkg/nugetlock/parse_testcase.go

```diff
@@ -0,0 +1,98 @@
+package nugetlock
```

Comment:
tried to rename with git mv, did not work.

Reply:
It's done.
https://github.com/aquasecurity/go-dep-parser/pull/42

Reply:
Rebase or merge the above PR